### DISCUSSION TITLE
Unit of nginxplus_upstream_server_response_time metric

### DISCUSSION THREAD

**Discussion by ryansabandal** [nginx/nginx-prometheus-exporter#726] [Q&A] [answered]
Hello,

Just want to ask what is the unit of the nginxplus_upstream_server_response_time metric? Is it in seconds or milliseconds?

Thanks!

**Comment by lucacome** [comment, marked as answer]
Hi @ryansabandal 

The time is in seconds.

**Reply by ryansabandal** [reply]
Thank you! Do you have any recommendations on how to visualize it in a way that it shows the trend?
